sdk/storage/azure-storage-blobs/test/perf/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ target_include_directories(
 
 if (MSVC)
     # allow msvc to use getenv()
-    target_compile_options(azure-storage-blobs-perf PUBLIC /wd4996)
+#    target_compile_options(azure-storage-blobs-perf PUBLIC /wd4996)
```

Comment:
Undo the change?

Reply:
Good catch, I'll remove the entire block (it's unnecessary and it causes active harm).